jenkins/spark-nightly-build.sh

```diff
@@ -17,6 +17,8 @@
 
 set -ex
 
+. jenkins/version-def.sh
```

Comment:
How was this ever working for scala2.13?

Reply:
For scala2.13 build, before your PR11308, this `.version-def.sh` below the cmd line "cd scala2.13"  works, because version-def.sh does NOT contains/refers any lines specifying relative path like 'pom=scala2.13/pom.xml' or 'build/get_buildvers.py'; so we can either run  `.version-def.sh` under the project's base dir, or under scala2.13

![image](https://github.com/user-attachments/assets/006ccdc3-ee67-42ba-b1ba-e5d3c3b14160)


But your change below in PR11308 requires to run `.version-def.sh` script under the project's base dir now!

![image](https://github.com/user-attachments/assets/03287242-c15c-4507-a3dd-4e16414a1149)

![image](https://github.com/user-attachments/assets/e5490b6f-0245-48f8-bda6-8c14dcc2a886)